components/runtime.rst

```diff
@@ -462,10 +458,23 @@ always using this ``ReactPHPRunner``::
 
 The end user will now be able to create front controller like::
 
-    require_once dirname(__DIR__).'/vendor/autoload_runtime.php';
+    use App\Runtime\ReactPhpRuntime;
+    use Psr\Http\Server\RequestHandlerInterface;
 
-    return function (array $context): SomeCustomPsr15Application {
-        return new SomeCustomPsr15Application();
+    $_SERVER['APP_RUNTIME'] = ReactPhpRuntime::class;
```

Comment:
Why does this need to be set here?

Reply:
It is for the application : by default it uses the SymfonyRuntime on the autoload_runtime, this is the way to choose our Runtime

Reply:
I am not sure if that is a good way to demo it in the example though. This would override whatever you would specify in the ENV of the webserver for example. Users might get the wrong impression.

Reply:
I will push a shorter version with all feedbacks, thanks @alcohol

Reply:
I push a new version without all this detail as explain just above to stof